client/src/containers/backend/People/Users/Edit.js

```diff
@@ -127,6 +127,13 @@ class UsersEditContainer extends PureComponent {
             name="attributes[lastName]"
             placeholder="Last Name"
           />
+          <FormContainer.UserClaims
```

Comment:
This needs to be moved outside of the attributes form. Having it above the save button suggests that the changes made require saving, which they do not. It should go below the save button. Look at page 10 of the backend UI comps to see header styles for breaking this out from the main form.